R/cache.R

```diff
@@ -256,6 +253,9 @@ build_cached_function <- function(cached_function) {
       }
     }
 
+    # argument 'force' may have a conflict with the to-be-cached function.
+    `_force` <<- if ("force" %in% names(call)) call[["force"]] else FALSE
```

Comment:
This isn't defined in a parent scope, so you are creating a global variable here I think.